### PR TITLE
Feature/frontend 2077 - e2e tests for new functionality

### PR DESCRIPTION
**Pull request by bnlb**
Resolves [FRONTEND-2077](https://algorithmia.atlassian.net/browse/FRONTEND-2077)

### CODE REVIEW

**Comment by bnlb** [comment]
This is deployed, so you can run `npm run e2e:cloud` and everything will pass.

File: _config.yml [hunk 1]
```diff
@@ -89,6 +89,7 @@ exclude:
     "shippable.yml",
     "_tmp",
     "vendor",
+    "e2e/",
```
Comment (langlk) [inline _config.yml]: 👍

**Comment by langlk** [comment]
It looks like tests are failing because the nav menu is starting out closed: see the recording for [this session](https://automate.browserstack.com/dashboard/v2/builds/80bc98e74b6f8d3720f68a59b970a06b7987a363/sessions/7a4ece4ab6f25bf84c24bbf63bcdd772dc4c39a6)

**Comment by bnlb** [comment]
> It looks like tests are failing because the nav menu is starting out closed: see the recording for [this session](https://automate.browserstack.com/dashboard/v2/builds/80bc98e74b6f8d3720f68a59b970a06b7987a363/sessions/7a4ece4ab6f25bf84c24bbf63bcdd772dc4c39a6)

Sorry about that. I only tested on OSX when wiring this up to make things faster. I bumped the width of the browser window up to 1250 earlier to account for this, but for some reason Windows isn't playing nicely while OSX is. I've skipped the tests in windows for the side nav toggle. I've verified side nav functionality is working as expected in Firefox and Chrome on Windows 10 on a cheap laptop I own for exactly this sort of thing. All tests are passing for me now.

**Pull request merged**